components/raftstore/src/store/peer.rs

```diff
@@ -3405,6 +3424,19 @@ where
         if self.pending_remove {
             return false;
         }
+        // When in the flashback state, we should not allow any other request to be
```

Comment:
do we need to reject reads?

Reply:
should check it in `pre_propose_raft_command`, returning false directly here would cause callback uninvoked.